src/main/groovy/uberdoc/Utils.groovy

```diff
@@ -0,0 +1,28 @@
+package uberdoc
+
+import java.lang.annotation.Annotation
+
+class Utils {
```

Comment:
i dont really like this utils class, but i havent found a better alternative yet.